File-Formats/XlsIO/Working-with-Template-Markers.md

```diff
@@ -497,7 +497,7 @@ By default, DataTable values will be filled in the worksheet as a string format.
 
  The following screenshot represents the input template which has a template marker.
 
-![Bind from DataTable Example](Working-with-Template-Markers_images/Working-with-Template_Markers_img3.jpeg)
+![Bind from DataTable Example](Working-with-Template-Markers_images/Bind_datatable_to_Marker_in_Excel_Spreadsheet_Templates.jpeg)
```

Comment:
Please check with product team regarding this
![image](https://user-images.githubusercontent.com/61148953/182146861-173a688f-d264-4752-af4b-6511b3641b31.png)

Reply:
confirmed with product team , the current change was fine
![image](https://user-images.githubusercontent.com/95337166/182301073-64cf9ddd-ae58-42e6-a6bd-0eead3e9ad8b.png)